### PR TITLE
Flashlight changes and fixes

### PR DESCRIPTION
**Pull request by Llywelwyn**
changes:
  - bugfix: "Flashlights now use much less power across the board. Most flashlights last ~26 minutes, with lanterns the exception at ~133 minutes, as they contain a larger cell."
  - tweak: "Flashlights now default to high brightness, and their charge is displayed on examine."
  - tweak: "Flashlight brightness can now be selected with alt-click - try turning it down if you need to conserve power!"
  - tweak: "Flashlights can now be placed into wall rechargers."
  - tweak: "Lanterns are now back to being small items, but have had their force reduced to compensate."
  - spellcheck: "Fixed various cell-related typos."

continuation of #16264 with permission from camo

tl;dr: flashlight power consumption was reduced to an eight what it was, but now they default to high brightness which uses twice as much power. everything will last ~4x longer, or twice/quadruple that if ppl reduce the brightness to med/low.

2 efficiency - penlights ~52min
1 efficiency - flashlights/heavy-duty ~26min 
0.8 efficiency - maglights ~20min
0.5 efficiency - lanterns start w/ a regular cell so they last ~133min

### CODE REVIEW

**Comment by Llywelwyn** [comment]
!review

**Review by alsoandanswer — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]
